docs/source/routing.rst

```diff
@@ -15,17 +15,18 @@ Weblocks:
 
 * use ``weblocks-navigation-widget:defroutes``
 
-* ``defroutes`` associates urls (as strings) to a widget::
+* ``defroutes`` associates URLs (as strings) to a widget::
 
 .. code-block:: common-lisp
 
-   (defroutes tasks-routes
+   (defroutes <routes-name>
```

Comment:
Why this name is better?

Reply:
it is supposed to be a placeholder for a name, so it should read generic (here we are at the general guidelines, not into the example), and I intent to highlight the `make-<…>` route contsructor.

Reply:
I'm asking because seen how some authors use this notation for class names. However, it seems weird to me.

Also, this is a code example and some people, reading the documentation, may decide that this is some notation for route class names. And they will use this in the real code.

It is better to use a name without `<>`

Reply:
done and rebased.